### PR TITLE
Feature/s3 c 3132 utapi v2 push metric

### PR DESCRIPTION
**Pull request by dora-korpar**
Updates pushMetric function and calls in each API to provide more information for pushMetric v2

### CODE REVIEW

File: lib/utapi/utilities.js [hunk 1]
```diff
@@ -206,8 +223,40 @@ function listMetrics(metricType) {
...
+
+    if (utapiVersion === 2) {
+        const utapiObj = {
+            operationId: action,
+            timestamp:
+            bucket,
+            location,
+            objectDelta: numberOfObjects,
+            sizeDelta: byteLength,
+            incomingBytes: -byteLength || newByteLength,
```
Comment (tmacro) [inline lib/utapi/utilities.js]: I'm not sure if these are right. `incomingBytes` and `outgoingBytes` represents the amount of data received or sent by the server during the operation.

**Review by tmacro — APPROVED** [review]

File: lib/utapi/utilities.js (hunk 1 above)
Comment (tmacro) [reply lib/utapi/utilities.js]: I meant to comment and not approve, but I can't figure out how to change it

**Review by naren-rajendran — APPROVED** [review]

File: lib/utapi/utilities.js (hunk 1 above)
Comment (miniscruff) [reply lib/utapi/utilities.js]: You can just re-review and it overrides your previous.

Comment (dora-korpar) [reply lib/utapi/utilities.js]: oh so they're not a running total? would the incomingBytes on a delete operation just be zero then?

Comment (tmacro) [reply lib/utapi/utilities.js]: yes & yes, the event reflects how an op would change the totals

**Comment by dora-korpar** [comment]
@bert-e approve

**Pull request merged**